ui/app/com/gu/recipeasy/views/recipe.scala.html

```diff
@@ -3,28 +3,38 @@
         recipeId: String,
         body: String,
         articleId: String,
-        shouldShowButtons: Boolean,
+        isEditable: Boolean,
         images: List[ImageDB],
         status: RecipeStatus)(implicit messages: play.api.i18n.Messages, request: RequestHeader)
 @import helper.CSRF
 @implicitFC = @{ b4.vertical.fieldConstructor }
 @link = @{ "https://www.theguardian.com/" + articleId }
 @pageTopMessage(status: RecipeStatus) = @{
-    status match {
-        case New        => "Creation, pass 1 of 3."
-        case Pending    => "Creation, pass 1 of 3."
-        case Curated    => "Verification, pass 2 of 3."
-        case Verified   => "Final Check, pass 3 of 3."
-        case Finalised  => "Final Check, pass 3 of 3."
-        case _ => ""
+    if(isEditable){
```

Comment:
FYI it's encouraged to have a space between `if` and the first bracket and a space between the last bracket and the brace. The same applies with else below. But it's a small thing, don't feel the need to change it in this PR.